src/client/components/TotalPriceCard/TotalPriceCard.component.js

```diff
@@ -7,9 +7,9 @@ const TotalPrice = ({ subTotal }) => {
   const grandTotal = subTotal > 0 ? subTotal + delivering : 0;
   return (
     <>
-      <aside className="wrapper">
+      <aside className="wrapperTotalPrice">
         <h3>Total</h3>
-        <div className="wrapper-content">
+        <div className="wrapperContentTotalPrice">
           <div className="row">
             <div>Subtotal</div>
             <div className="text-right">{subTotal.toFixed(2)} DKK</div>
```

Comment:
```suggestion
            <div>{subTotal.toFixed(2)} DKK</div>
```

You are adding in the css the property of `justify-content: space-between; `to the row. What this does is that it distributes the elements within the total width of the container, distributing them within the whole width. In other words, you are aligning the amounts within `row` twice.

Reply:
While in line 22, it is ok to keep the css for `'text-right'`, but maybe a better name would be` total-amount`

Reply:
Made the changes as per your review :)